### PR TITLE
Structured clone

### PR DESCRIPTION
**Pull request by CheloXL**
Fixes cloning RegEx in Chrome48.
Adjusted tests.

### CODE REVIEW

File: polyfills/structuredClone/polyfill.js [hunk 1]
```diff
@@ -0,0 +1,270 @@
...
+			return as([TYPE, {name: typeName, message: value.message}], value);
+		}
+
+		return pair;
+	}
+
+	function serialize(value, options) {
+		var _ = [];
+		options = options || {};
+		return serializer(!(options.json || options.lossy), !!options.json, new Map(), _)(value), _;
```
Comment (zloirock) [inline polyfills/structuredClone/polyfill.js:263]: This is a polyfill. It's harmful to allow such options.

Comment (CheloXL) [reply polyfills/structuredClone/polyfill.js]: Done. Options removed.

**Review by romainmenke — APPROVED** [review]
CI all green 🎉 

Thank you @CheloXL and everyone for the feedback and help with this one!

**Pull request merged**